example/api/index.js

```diff
@@ -35,8 +35,8 @@ app.post('/request-register', (req, res) => {
     res.send(challengeResponse);
 });
 
-app.post('/register', (req, res) => {
-    const { key, challenge } = parseRegisterRequest(req.body);
+app.post('/register', async (req, res) => {
```

Comment:
Does express actually support this? I was under the impression we had to use something like this:

```
const asyncHandler = fn => (req, res, next) =>
  Promise
    .resolve(fn(req, res, next))
    .catch(next)
```

http://www.acuriousanimal.com/2018/02/15/express-async-middleware.html